docs/faq.md

```diff
@@ -36,20 +36,18 @@ of course constantly aim to improve the bot but it will _always_ be a
 gamble, which should leave you with modest wins on monthly basis but
 you can't say much from few trades.
 
-### I’d like to change the stake amount. Can I just stop the bot with /stop and then change the config.json and run it again?
+### I’d like to make changes to the config. Can I do that without having to kill the bot?
 
-Not quite. Trades are persisted to a database but the configuration is
-currently only read when the bot is killed and restarted. `/stop` more
-like pauses. You can stop your bot, adjust settings and start it again.
+Yes. You can edit your config, use the `/stop` command in Telegram, followed by `/reload_config` and the bot will run with the new config.
 
 ### I want to improve the bot with a new strategy
 
 That's great. We have a nice backtesting and hyperoptimization setup. See
 the tutorial [here|Testing-new-strategies-with-Hyperopt](bot-usage.md#hyperopt-commands).
 
-### Is there a setting to only Buys the coins being held and not perform anymore Buys?
+### Is there a setting to only SELL the coins being held and not perform anymore BUYS?
 
-You can use the `/stopbuy` to prevent future buys, followed `/forcesell all` (sell all open trades) command from Telegram.
+You can use the `/stopbuy` command in Telegram to prevent future buys, followed by `/forcesell all` (sell all open trades).
```

Comment:
I wouldn't call the current version incorrect - it's just formatted differently.